config/manager/manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - /manager
         args:
         - --enable-leader-election
-        image: quay.io/opentelemetry/opentelemetry-operator:master
+        image: controller
```

Comment:
This is reverting a previous change. I didn't notice that there is a `kustomization` that replaces the image name.

Reply:
Ok thanks.